### ISSUE TITLE
Accept credit card payments

### ISSUE THREAD

**Issue by ccverg** [bountysource/core#486]
From support:

> Though, one more thing. You should make it possible to contribute without requiring a Google Wallet or Paypal account. According to [1] it should be possible if you are a Premier or Business account holder with paypal. This time around I had to create a paypal account and then delete it after my payment. Next time I might not find it worthwhile. I just want to pay with my VISA.
> 
> [1] https://www.paypal.com/cgi-bin/webscr?cmd=p/mer/WAX_landing-outside
##

**Comment by avodonosov** [comment]
Yes, For me PayPal is a show stopper. I just wanted to donate to a project but am unable because of this. Searching about Bountysource payment methods I found this issue. I would supported it by a bounty, but it's a circular problem - I can't pay with PayPal

**Comment by lmamane** [comment]
Adding my vote to this long-standing issue. Any reason Bountysource Salt accepts stripe / credit cards but not straight Bountysource?

**Comment by lmamane** [comment]
With today's TOS change of expiring two-year old bounties and Bountysource keeping the money, I lost interest in using Bountysource, even if you added non-Paypal options.
